Khan/wonder-blocks
# WB-863: Allow overlapping the popper inside Modals

Summary:

- Fixes the Popper instances by setting `position: fixed` to avoid adding
extra scrolls when we use these components (Dropdown, Tooltip, Popover)
inside a Modal.
- Dropdown: Adds the `tether` key to the `preventOverflow` modifier to
  be able to calculate the overlap correctly (if there's one) inside
  modals.

Issue: https://khanacademy.atlassian.net/browse/WB-863

Test plan:

- Navigate to the Dropdown examples (especially `DropdownInModal`).
- Resize the browser's height and verify that the popper (dropdown
  options container) is displayed completely.

https://user-images.githubusercontent.com/843075/136284317-c725ef86-d086-4fee-90e3-af01913ef6f9.mov
